backend/template-generation/test/test-template-generation.js

```diff
@@ -0,0 +1,17 @@
+let converter = require('../generate-source-file.js');
+let data = {
```

Comment:
We will have to figure out precisely what data we want from LinkedIn

Reply:
This is just for testing the functionality of the script, once we have sample data from LinkedIn we can create tests for that as well.